cmd/plugin/rpaasv2/cmd/status.go

```diff
@@ -35,8 +35,6 @@ var statusCmd = &cobra.Command{
 		status.service = cmd.Flag("service").Value.String()
 		status.instance = cmd.Flag("instance").Value.String()
 		status.prox = proxy.New(status.service, status.instance, "GET", &proxy.TsuruServer{})
-		// status.prox = &proxy.Proxy{ServiceName: status.service, InstanceName: status.instance, Method: "GET"}
```

Comment:
You can safely drop these.

Reply:
weird, this comment does not show on my local branch and when i run git diff on the remote it doesn't show the difference either...